docs/docs/associations.md

```diff
@@ -42,30 +42,30 @@ User.belongsTo(Company); // Will add company_uuid to user
 In cases where `as` has been defined it will be used in place of the target model name.
 
 ```js
-var User = this.sequelize.define('User', {/* attributes */})
-  , UserRole  = this.sequelize.define('UserRole', {/* attributes */});
+var User = this.sequelize.define('user', {/* attributes */})
+  , UserRole  = this.sequelize.define('userRole', {/* attributes */});
 
-User.belongsTo(UserRole, {as: 'Role'}); // Adds RoleId to user rather than UserRoleId
+User.belongsTo(UserRole, {as: 'role'}); // Adds roleId to user rather than userRoleId
 ```
 
 In all cases the default foreign key can be overwritten with the `foreignKey` option.
 When the foreign key option is used, Sequelize will use it as-is:
 
 ```js
-var User = this.sequelize.define('User', {/* attributes */})
-  , Company  = this.sequelize.define('Company', {/* attributes */});
+var User = this.sequelize.define('user', {/* attributes */})
+  , Company  = this.sequelize.define('company', {/* attributes */});
 
 User.belongsTo(Company, {foreignKey: 'fk_company'}); // Adds fk_company to User
 ```
 
 #### Target keys
 
-By default the target key for a belongsTo relation will be the target primary key. To override this behavior, use the `targetKey` option.
+Its the column that will be referred from **target model**. By default the target key for a belongsTo relation will be the target model's primary key. To override this behavior, use the `targetKey` option.
```

Comment:
Not sure this explanation makes it more clear, maybe:

```
The target key is the column on the target model that the foreign key column on the source model points to. By default the target key for a belongsTo relation will be the target models primary key. To define a custom column, use the `targetKey` option.
```

`model's` => `models` (otherwise it reads as `model is primary key`)